test/integration/Cover/migrateCoverAndSubmit.js

```diff
@@ -0,0 +1,66 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+const { getCoverSegment } = require('../../unit/IndividualClaims/helpers');
+
+const { parseEther } = ethers.utils;
+
+function zeroPadRight(bytes, length) {
+  return new Uint8Array(length).fill(0).map((x, i) => bytes[i] || x);
+}
+
+describe('createStakingPool', function () {
+  let coverOwner;
+  let coverId;
+  before(async function () {
+    const ETH = zeroPadRight(Buffer.from('ETH'), 4);
+    const { qd, tk: token } = this.contracts;
+    coverOwner = this.accounts.members[1];
```

Comment:
create some spaces here between contract initialization, account initialization and the rest of the code.

Use more spaces when you write test code to make things easier to read and group things logically.

For example see how these tests are written:

https://github.com/NexusMutual/smart-contracts/blob/d737f8f28d9797c1a2a415d51a8337b1b1e5bdf1/test/unit/IndividualClaims/redeemClaimPayout.js#L56